From f5929b6aa1f92b8572e516e77253c9849ae149f3 Mon Sep 17 00:00:00 2001
From: Portisch <hugo.portisch@yahoo.de>
Date: Wed, 6 Dec 2023 09:14:20 +0100
Subject: [PATCH 2/2] Partial revert "Remove remaining dependencies to Estuary
 skin"

This reverts commit b58df0e0c3b149d5613c4ac06f213ea627a440ed.

Use kodi color theme.
---
 .../service-CoreELEC-Settings-mainWindow.xml  | 24 +++++++++----------
 .../service-CoreELEC-Settings-wizard.xml      |  2 +-
 2 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/skins/Default/1080i/service-CoreELEC-Settings-mainWindow.xml b/skins/Default/1080i/service-CoreELEC-Settings-mainWindow.xml
index e257694..c06733e 100644
--- a/skins/Default/1080i/service-CoreELEC-Settings-mainWindow.xml
+++ b/skins/Default/1080i/service-CoreELEC-Settings-mainWindow.xml
@@ -1142,8 +1142,8 @@
             <top>940</top>
             <width>48</width>
             <height>24</height>
-            <texturefocus colordiffuse="FF12A0C7">arrowdown.png</texturefocus>
-            <texturenofocus colordiffuse="FF12A0C7">arrowdown.png</texturenofocus>
+            <texturefocus colordiffuse="button_focus">arrowdown.png</texturefocus>
+            <texturenofocus colordiffuse="button_focus">arrowdown.png</texturenofocus>
             <animation effect="zoom" start="100,0" end="100,100" center="auto" time="200" delay="10">VisibleChange</animation>
             <animation effect="zoom" start="100,0" end="100,100" delay="500" center="auto" time="200">WindowOpen</animation>
             <animation effect="zoom" start="100,100" end="100,0" center="auto" time="200">WindowClose</animation>
@@ -1156,8 +1156,8 @@
             <top>105</top>
             <width>48</width>
             <height>24</height>
-            <texturefocus colordiffuse="FF12A0C7" flipy="true">arrowdown.png</texturefocus>
-            <texturenofocus colordiffuse="FF12A0C7" flipy="true">arrowdown.png</texturenofocus>
+            <texturefocus colordiffuse="button_focus" flipy="true">arrowdown.png</texturefocus>
+            <texturenofocus colordiffuse="button_focus" flipy="true">arrowdown.png</texturenofocus>
             <animation effect="zoom" start="100,0" end="100,100" center="auto" time="200" delay="10">VisibleChange</animation>
             <animation effect="zoom" start="100,0" end="100,100" delay="500" center="auto" time="200">WindowOpen</animation>
             <animation effect="zoom" start="100,100" end="100,0" center="auto" time="200">WindowClose</animation>
@@ -1172,8 +1172,8 @@
             <top>940</top>
             <width>48</width>
             <height>24</height>
-            <texturefocus colordiffuse="FF12A0C7">arrowdown.png</texturefocus>
-            <texturenofocus colordiffuse="FF12A0C7">arrowdown.png</texturenofocus>
+            <texturefocus colordiffuse="button_focus">arrowdown.png</texturefocus>
+            <texturenofocus colordiffuse="button_focus">arrowdown.png</texturenofocus>
             <animation effect="zoom" start="100,0" end="100,100" center="auto" time="200" delay="10">VisibleChange</animation>
             <animation effect="zoom" start="100,0" end="100,100" delay="500" center="auto" time="200">WindowOpen</animation>
             <animation effect="zoom" start="100,100" end="100,0" center="auto" time="200">WindowClose</animation>
@@ -1186,8 +1186,8 @@
             <top>105</top>
             <width>48</width>
             <height>24</height>
-            <texturefocus colordiffuse="FF12A0C7" flipy="true">arrowdown.png</texturefocus>
-            <texturenofocus colordiffuse="FF12A0C7" flipy="true">arrowdown.png</texturenofocus>
+            <texturefocus colordiffuse="button_focus" flipy="true">arrowdown.png</texturefocus>
+            <texturenofocus colordiffuse="button_focus" flipy="true">arrowdown.png</texturenofocus>
             <animation effect="zoom" start="100,0" end="100,100" center="auto" time="200" delay="10">VisibleChange</animation>
             <animation effect="zoom" start="100,0" end="100,100" delay="500" center="auto" time="200">WindowOpen</animation>
             <animation effect="zoom" start="100,100" end="100,0" center="auto" time="200">WindowClose</animation>
@@ -1202,8 +1202,8 @@
             <top>940</top>
             <width>48</width>
             <height>24</height>
-            <texturefocus colordiffuse="FF12A0C7">arrowdown.png</texturefocus>
-            <texturenofocus colordiffuse="FF12A0C7">arrowdown.png</texturenofocus>
+            <texturefocus colordiffuse="button_focus">arrowdown.png</texturefocus>
+            <texturenofocus colordiffuse="button_focus">arrowdown.png</texturenofocus>
             <animation effect="zoom" start="100,0" end="100,100" center="auto" time="200" delay="10">VisibleChange</animation>
             <animation effect="zoom" start="100,0" end="100,100" delay="500" center="auto" time="200">WindowOpen</animation>
             <animation effect="zoom" start="100,100" end="100,0" center="auto" time="200">WindowClose</animation>
@@ -1216,8 +1216,8 @@
             <top>105</top>
             <width>48</width>
             <height>24</height>
-            <texturefocus colordiffuse="FF12A0C7" flipy="true">arrowdown.png</texturefocus>
-            <texturenofocus colordiffuse="FF12A0C7" flipy="true">arrowdown.png</texturenofocus>
+            <texturefocus colordiffuse="button_focus" flipy="true">arrowdown.png</texturefocus>
+            <texturenofocus colordiffuse="button_focus" flipy="true">arrowdown.png</texturenofocus>
             <animation effect="zoom" start="100,0" end="100,100" center="auto" time="200" delay="10">VisibleChange</animation>
             <animation effect="zoom" start="100,0" end="100,100" delay="500" center="auto" time="200">WindowOpen</animation>
             <animation effect="zoom" start="100,100" end="100,0" center="auto" time="200">WindowClose</animation>
diff --git a/skins/Default/1080i/service-CoreELEC-Settings-wizard.xml b/skins/Default/1080i/service-CoreELEC-Settings-wizard.xml
index 9aad8c1..ee36264 100644
--- a/skins/Default/1080i/service-CoreELEC-Settings-wizard.xml
+++ b/skins/Default/1080i/service-CoreELEC-Settings-wizard.xml
@@ -66,7 +66,7 @@
                 <top>575</top>
                 <width>1050</width>
                 <height>3</height>
-                <texture colordiffuse="FF12A0C7" border="2">separator-grey.png</texture>
+                <texture colordiffuse="button_focus" border="2">separator-grey.png</texture>
             </control>
             <control type="group" id="1398">
                 <visible>String.IsEmpty(Control.GetLabel(1390))</visible>
-- 
2.43.0

